wurst/objediting/SharedEnums.wurst

```diff
@@ -0,0 +1,41 @@
+package SharedEnums
```

Comment:
Why not name the file `Race.wurst`?

Reply:
At one point I had more than one Enum in the package (armor type for the set armor defense type effect), but I ended up only sharing race. Perhaps in the future this wouldn't be a bad space for other shared enums to go? Or I can rename it to Race.

Reply:
It would be good to have shared/common objediting stuff in one packge, but the name "SharedEnums" isn't good imo.